test/unittests/constexpr_vector_test.cpp

```diff
@@ -27,6 +27,14 @@ TEST(constexpr_vector, size)
     EXPECT_EQ(*v3.data(), 1);
 }
 
+TEST(constexpr_vector, subscript)
+{
+    constexpr constexpr_vector<int, 4> v = {1, 2, 3};
```

Comment:
Does `<int, 4>` mean there are 4 members, the last being implicitly 0?

Reply:
No. The `4` is max capacity. Size is still 3 here. See https://github.com/wasmx/fizzy/pull/656.